components/forms/form-builder/FormBuilder.tsx

```diff
@@ -77,6 +77,7 @@ type Section = {
 export type Form = {
   name: string;
   pages: Page[];
+  declarations: { name: string; label: string }[];
```

Comment:
You don't think making this a proper type is worth doing at this stage? (Also, any inclination to update that lovely README 😈 ?)

Reply:
@ReubenRobertsHEE Thanks for reviewing on a Fri pm 💪.
RE the type, the whole `declarations` prop is a bit of a WIP at the moment; was umming and ahhing on whether or not to make this into something that could handle more than just the submission declarations but then pictured your wagging finger...A similar excuse for the README update too and I'm sticking to it 😁 .
Happy to push some changes with this commit though if it helps to get this one over the line.

Reply:
Good point: I should appreciate these wafer-thin PRs for what they are! 😍